power_grid_model_c/power_grid_model/include/power_grid_model/common/exception.hpp

```diff
@@ -121,7 +121,7 @@ class InvalidRegulatedObject : public PowerGridError {
 class AutomaticTapCalculationError : public PowerGridError {
   public:
     AutomaticTapCalculationError(ID id) {
-        append_msg("Automatic tap changing regulator is at LV side is not supported. Found at id" +
+        append_msg("Automatic tap changing regulator with tap_side at LV side is not supported. Found at id" +
```

Comment:
if you change this exception, you also need to change the message in the python error